dev/package-examples/nginx-1.2.0/dataset/stubstatus/docs/README.md

```diff
@@ -7,3 +7,5 @@ An example event looks as following:
 ```
 {{ > data.json }}
 ```
+
+The stubstatus input support the [standard http config](https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-metricbeat.html#module-http-config-options) options.
```

Comment:
@exekias @andresrc @ph @Titch990 @sorantis This is one of this odd places where we probably have to link from the package docs to the Beat specific docs. Probably ok for the first release but something we should think on how to solve long term.

Note: Just put this in here as an example, the docs are not complete here.